src/common/components/layouts/pages/inspect/toolbar/Toolbar.scss

```diff
@@ -40,3 +40,39 @@
     text-overflow: ellipsis;
   }
 }
+
+@media (max-width: 1070px) {
+  .toolbar {
+     section:not(:first-of-type) {
+       flex-grow: 2;
+    }
+  }
+}
+
+@media (max-width: 880px) {
```

Comment:
Let's set the max-width to ~895px in order to prevent the "Back to summary" link wrapping in screen width interval [880px; 895px]